doc/TOR.md

```diff
@@ -255,48 +265,51 @@ Other nodes will not be able to `connect` to you unless you communicate them how
+Once the .onion addresses have been created with the procedures [oulined above](#creation-of-an-hidden-service-for-a-persistent-.onion-address), 
+the node is already reachable at the .onion address.
 
-External users can connect to this node by TOR V2 and V3 and a random V2 until next tor release, then also (V3 randomly).
+To make your external .onion addresses public you add: `--announce-addr=.onionAddressV2:port --announce-addr=.onionAddressV3:port` to the options to publish your IP number.
 
-The Persistent addresses can be created with the steps outlined above.
+#### Case #7 c-lightning has no public IP address and a fixed Tor V3 service address and fixed Tor V2 service  address a 3rd non persisten V2 address
 
-You are not obliged to announce the non persistent V2 address but if want to do it:
+External users can connect to this node by Tor V2 and V3 and a random V2 until next tor release, then also (V3 randomly).
 
-`--addr=autotor:<torservice_ip:port>`
+The Persistent addresses can be created with the steps [outlined above](#creation-of-an-hidden-service-for-a-persistent-.onion-address).
 
-and also you must specify `--tor-service-password=yourpassword` (not the hash) to access the
+To create your non-persistent Tor address, add 
+`--addr=autotor:127.0.0.1:9051` if you want to announce it or 
+`--bind-addr=autotor:127.0.0.1:9051` if you don't want to announce it.
 
-tor service at 9051 If you have protected them with the password (no additional options if
-they are protected with a cookie file. See above.).
+Also you must specify `--tor-service-password=yourpassword` (not the hash) to access the 
+Tor service at 9051 If you have protected them with the password (no additional options if
+they are protected with a cookie file. [See above](#creation-of-an-auto-service-for-non-persistent-.onion-addresses)).
 
 To make your external .onion address (V2 and V3) public you add: `--bind-addr=yourInternalIPAddress:port --announce-addr=your.onionAddressV2:port --announce-addr=your.onionAddressV3:port`
 
-NOTE: if you want both of them public you can repeat the --announce-addr option. If your node is NOT inside an internal network you can use `--addr=external` instead.
+#### Case #8 	c-lightning has a public IP address and no Tor addresses
 
+The external address is communicated by the 
+`--bind-addr=internalIPAddress:port --announce-addr=yourexternalIPAddress:port`
+or `--addr=yourexternalIPAddress:port` if the node is not inside an internal network.
 
-#### Case #8 	c-lightning has a public IP address and no TOR hidden service address,
-
-The external address is communicated by the `--announce-addr=yourexternalIPAddress:port`
-
-but can connect to any V4/6 ip address via a IPV4/6 socks 5 proxy by specifing 
+The node can connect to any V4/6 ip address via a IPV4/6 socks 5 proxy by specifing 
 `--proxy=127.0.0.1:9050 --always-use-proxy`.
 
-References
+
```

Comment:
I like the the separate References header, though...

Reply:
Thank you. 
ACK on my part for everything.

I will make some attempt for the "references" to hide it in markdown or to make the links visible.